Manager/MimeTypeManager.php

```diff
@@ -100,4 +100,17 @@ public function getImageMimeTypesChoices(): array
 
         return array_combine($extensions, $extensions);
     }
+
+    /**
+     * @return array
+     */
+    public function getAllMimeTypes(): array
+    {
+        $mimeTypes = [];
+        foreach ($this->mimeTypes as $item) {
+            $mimeTypes[] = $item;
+        }
+
+        return array_merge([], ...$mimeTypes);
```

Comment:
`return return array_merge([], ...$this->mimeTypes);` ça fonctionne pas ?

Reply:
Ce n'est pas exactement la même chose, le array_merge([], ...$this->mimeTypes) n'aime pas trop avec des clefs de tableau style "sherlockode_advanced_content.mime_type_group.text_file"

Le foreeach créé un tableau basique et là ça marche